### PR TITLE
Change /communities to be the landing page

### PR DESCRIPTION
**Pull request by Oksamies**
Change links to landing page to be to community list view
Redirect /communities to / when enabled

 refs TS-1828

### CODE REVIEW

File: apps/cyberstorm-nextjs/.env.development [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Values prefixed with NEXT_PUBLIC_ are available in browser, with all
 # the security implications that entails.
+NEXT_PUBLIC_PRODUCTION_PAGE_STRUCTURE=
```
Comment (anttimaki) [inline apps/cyberstorm-nextjs/.env.development:3]: I second what Mythic mentioned in Discord, I think this is overkill.

File: apps/cyberstorm-nextjs/middleware.ts [hunk 2]
```diff
@@ -24,6 +24,12 @@ export async function middleware(request: NextRequest) {
   const req = request;
   const response = NextResponse.next();
 
+  if (process.env.NEXT_PUBLIC_PRODUCTION_PAGE_STRUCTURE) {
+    if (request.nextUrl.pathname.startsWith("/communities")) {
```
Comment (anttimaki) [inline apps/cyberstorm-nextjs/middleware.ts:28]: This blocks the access to all subpages under `/communities`. Sure, we don't have them yet, but wouldn't hurt to use exact matching here either.

**Review by anttimaki — CHANGES_REQUESTED** [review]
Looks thorough, e.g. checking the footer and links which I forgot to mention in the task 👍🏻

**Review by anttimaki — APPROVED** [review]
The changes are good but the commit message is now blatantly wrong. Feel free to merge without another review once that's fixed.

**Comment by Oksamies** [comment]
Ah, nice catch, didn't even realize that

**Pull request merged**